docs/designs/schema-validation.md

```diff
@@ -17,12 +17,12 @@ Below are the attributes we use with the equivalent keywords in json schema.
 | MaxLengthAttribute | maxItems | An array or string instance is valid against if its size is less than, or equal to, the value of this keyword. | System.ComponentModel.DataAnnotations |
 | RegularExpressionAttribute | pattern | A string instance is considered valid if the regular expression matches the instance successfully. | System.ComponentModel.DataAnnotations |
 | JsonRequiredAttribute | required | An object instance is valid against this keyword if every required property exists in the instance. Notice that we could not use `RequiredAttribute` here since we execute the validation for each json node with the defined validation attribute, if the required field is missing from json object, we would not execute the requried validation. So we use `JsonRequriedAttribute` from Newtonsoft.Json, which would take effect during the deserialization. | Newtonsoft.Json |
-| JsonExtensionDataAttribute | additionalProperties | If no property in the type contains `JsonExtensionDataAttribute`, additional properties are not allowed and unknown field warning will be added if any found in the instance. Otherwise, no warning. | Newtonsoft.Json |
+| `sealed` modifier | additionalProperties | If the type class is sealed , additional properties are not allowed and unknown field warning will be added if any found in the instance. Otherwise, no warning. | Newtonsoft.Json |
```

Comment:
the implemtation is still using `JsonExtensionDataAttribute`, sealed is just to generate JSON schema